packages/flutter/test/widgets/selectable_text_test.dart

```diff
@@ -192,22 +189,32 @@ void main() {
 
   testWidgets('has expected defaults', (WidgetTester tester) async {
     await tester.pumpWidget(
-      const MediaQuery(
-        data: MediaQueryData(devicePixelRatio: 1.0),
-        child: Directionality(
-          textDirection: TextDirection.ltr,
-          child: SelectableText('selectable text'),
-        ),
+      boilerplate(
+          child: const SelectableText('selectable text'),
       ),
     );
 
-    final SelectableText selectableText =
-    tester.firstWidget(find.byType(SelectableText));
+    final SelectableText selectableText = tester.firstWidget(find.byType(SelectableText));
     expect(selectableText.showCursor, false);
     expect(selectableText.autofocus, false);
     expect(selectableText.dragStartBehavior, DragStartBehavior.start);
     expect(selectableText.cursorWidth, 2.0);
     expect(selectableText.enableInteractiveSelection, true);
+
+    RenderBox renderBox = tester.renderObject(find.byType(SelectableText));
```

Comment:
Let's separate this part into a different test. The original test is to test it has expected default values, while this part is testing the box size changes based on textScaleFactor.

Reply:
done